src/main/scala/com/example/solution/dto/reservation.scala

```diff
@@ -0,0 +1,24 @@
+package com.example.solution.dto
+
+import com.example.solution.dto.screening.ScreeningData
+
+import java.time.LocalDateTime
+
+object reservation {
+
+  case class PickedSeat(row : Int, seatInRow : Int, ticketType : String)
+
+  case class ReservationData(
+                            screeningId : Int,
+                            pickedSeats: List[PickedSeat],
+                            name : String,
```

Comment:
why you didn't reuse the previously defined types for name and surname?

Reply:
I guess I simply forgot about them.